src/main/js/src/store/actions/registration.js

```diff
@@ -1,66 +1,96 @@
 import * as actionTypes from './actionTypes';
 import axios from '../../axios';
-import i18next from 'i18next';
 
-import { firstCharToUpper } from '../../util/util';
-
-export const fetchExamLocations = () => {
-  const locations = ['Pernäjä', 'Hämeenlinna', 'Jämsä', 'Korso', 'Ankkalinna'];
+export const fetchExamSessions = () => {
   return dispatch => {
-    dispatch(fetchExamLocationsStart());
-    // TODO: get actual locations from api
-    // axios.get('/yki/api/exam-sessions')
-    dispatch(fetchExamLocationsSuccess(locations));
+    dispatch(fetchExamSessionsStart());
+    axios
+      .get('/yki/api/exam-sessions')
```

Comment:
endpoint is /yki/api/exam-session

Reply:
Oops! Will fix on next branch while improving visuals and hiding exam sessions in collapsible element.